common/document_parser/lib/entities.py

```diff
@@ -1,12 +1,60 @@
 from gamechangerml.src.utilities.text_utils import simple_clean, utf8_pass
 import gamechangerml.src.utilities.spacy_model as spacy_
+from common.document_parser.lib.entities_utils import *
 import collections
+import os
+from gamechangerml import DATA_PATH
 
 spacy_model = spacy_.get_lg_nlp()
-
+entities_dict = make_entities_dict(os.path.join(DATA_PATH,"features/GraphRelations.xls"))
 
 def extract_entities(doc_dict):
-    # get entities in each page. then, check if the entities are mentioned in the paragraphs and append
+    # Utilizes GraphRelations.xlsx in gamechangerml to find gold standard entities within each page's text. Then, check
+    # if the entities are mentioned in the paragraphs and append to paragraph  metadata.
+    page_dict = {}
+    for page in doc_dict["pages"]:
+        page_text = utf8_pass(page["p_raw_text"])
```

Comment:
I know I had this step in my code, but I think if we're using the pre-parsed JSONs the `p['text']` field has already had the cleaning function applied, so using that instead of cleaning raw text might save time

Reply:
Gotcha, ok! The previous spacy-based function (can be seen at the bottom of this file) used this same cleaning logic on the `p_raw_text`, so I was keeping things consistent, but can change if desired